packages/devextreme/js/__internal/ui/form/m_form.layout_manager.ts

```diff
@@ -265,15 +324,17 @@ class LayoutManager extends Widget<LayoutManagerProperties> {
     return result;
   }
 
-  _isAcceptableItem(item) {
-    const itemField = item.dataField || item;
+  _isAcceptableItem(item: Item | string): boolean {
+    const itemField = (item as SimpleItem).dataField ?? (item as string);
```

Comment:
Safer to use type guard here
```suggestion
  const itemField = isSimpleItem(item) ? item.dataField : item;
```
or
```suggestion
    const itemField = isString(item) ? item : item.dataField;
```